### PR TITLE
Service operators can add notes to claims

### PR DESCRIPTION
**Pull request by tekin**
This adds a simple notes feature that allows service operators to add notes against claims:

Screenshot 2020-03-30 at 17 08 31

### CODE REVIEW

File: spec/models/note_spec.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require "rails_helper"
+
+RSpec.describe Note, type: :model do
```
Comment (robbiepaul) [inline spec/models/note_spec.rb]: Do we need this file if there are no specs for the model?

File: spec/requests/admin_notes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "admin/notes controller" do
+  let(:claim) { create(:claim, :submitted) }
+
+  describe "admin/notes#index" do
+    it "list the notes on a claim when the user is a service operators" do
```
Comment (robbiepaul) [inline spec/requests/admin_notes_spec.rb]:
```suggestion
    it "list the notes on a claim when the user is a service operator" do
```

**Comment by tekin** [comment]
@robbiepaul good point re: the form. I imagine most claims will have very few notes, but I'll check in with Rich and Mark, seee what they say.

File: spec/models/note_spec.rb (hunk 1 above)
Comment (tekin) [reply spec/models/note_spec.rb]: Not really, but don't think it costs anything either. It also means it's there ready for someone if/when they need to add specs to the model.

**Comment by tekin** [comment]
Cheers @robbiepaul, word is that service operators should see the most important info first, so form stays at the bottom, plus I've reversed the order of the claims so that the most recent one comes at the top.

**Pull request merged**